proposals/sample-proposal/README.md

```diff
@@ -1,5 +1,36 @@
+| Title | Author | Date |
+| --- | --- | --- |
+| Title of proposal | `Your name` | today |
+
 # PROPOSAL NAME
 
-## Links:
-Discussion: [Discourse Forum](https://forum.theodinproject.com/c/sprints)
-Other Links
+## Summary
+
+Brief explanation of the proposal.
+
+## Motivation
+
+Explain the motivation behind this proposal. What is the purpose of this change or implementation? What does it solve? What is the expected outcome of this proposal?
+
+Make sure to express the motivation of this proposal in the best way you can so that if the change or implementation that is suggested here is not accepted, this motivation can be used to come up with an alternative solution.
+
+## Implementation
+
+Details and specifics about the solution being proposed, how will this be implemented? Include examples or diagrams that can help explain. If it is a change in flow or management specify what it should be changed to and how will that look exactly.
+
+## Drawbacks
+
+Are there any drawbacks to this proposal? consider the following:
+
+- Complexities or time consuming.
+- Amount of people needed to develop or availability of your own time.
+- Impact of the current state.
+- Will users need to change their flow? in what way?
+
+## Alternatives
+
+Are there any alternative implementations to this? What else can be done?
+
+## Additional
+
+This can be any unanswered questions, things that still need to considered or solved about the implentation, any reference links to the initial ideation such as a [Discourse Forum](https://forum.theodinproject.com/c/sprints), discussions or anything else.
```

Comment:
Can we add an additional "status" section?

```
## Status
- [ ] Proposed
- [ ] Accepted
- [ ] Planned
- [ ] Working on - [Link to Project Board](https;//github.com)
- [ ] Validating
- [ ] Complete
```

I think it could be a good way to track the progress of each RFC